### PR TITLE
Add icon for pinned tabs in Safari

### PR DESCRIPTION
**Pull request by vekunz**
The browser Safari supports pinned tabs since version 9, but you need a special icon for this tab. Otherwise only a default icon is shown, and not the favicon.

### CODE REVIEW

**Comment by Sumafu** [comment]
I don't know, why some of the test failed. But some of them doesn't failed because of my changes. For example the tests with the screenshots could not fail, because I changed nothing on the UI.

File: plugins/CoreHome/images/applePinnedTab.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 16 16">
+
+<g transform="translate(0 500) scale(0.1 -0.1)" fill="#000000" stroke="none">
```
Comment (tsteur) [inline plugins/CoreHome/images/applePinnedTab.svg]: The color should be ideally `#d4291f`

**Comment by tsteur** [comment]
I'm pretty sure the failing tests are not related to your change. I commented on the color and will try to test it in the next days

**Comment by tsteur** [comment]
Thanks for creating the PR! 👍

File: plugins/CoreHome/templates/_applePinnedTabIcon.twig [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {% if isCustomLogo and customFavicon is defined and customFavicon %}
 {% else %}
-	<link rel="mask-icon" href="plugins/CoreHome/images/applePinnedTab.svg" color="#C81319">
+	<link rel="mask-icon" href="plugins/CoreHome/images/applePinnedTab.svg" color="#d4291f">
```
Comment (Sumafu) [inline plugins/CoreHome/templates/_applePinnedTabIcon.twig]: The svg image must be completely black. The color is written into in the HTML-Head. Here some more information about the Tab Icon: [Apple Developer Library](https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html)

**Comment by Sumafu** [comment]
I changed the color.

File: plugins/CoreHome/templates/_applePinnedTabIcon.twig (hunk 2 above)
Comment (tsteur) [inline plugins/CoreHome/templates/_applePinnedTabIcon.twig]: 👍  thx for the link

**Comment by tsteur** [comment]
I have not forgotten you. I will merge in a couple of days as I first need to merge another big PR to avoid merge issues.

**Comment by tsteur** [comment]
Thanks @vekunz Just tested it in Safari and works great!

**Pull request merged**